### ISSUE TITLE
Duplicated Output When Using <body> Tag

### ISSUE THREAD

**Issue by SirKaleeb** [blitz-js/blitz#2190]
### What is the problem?

Good afternoon!

Discovered this bug when using the `<body>` tag when returning the JSX/TSX.

### Paste all your error logs here:

No errors were thrown.

### Paste all relevant code snippets here:

```
const Home: BlitzPage = () => {
  return (
    <>
      <Head>
        <title>Pollls</title>
      </Head>

      <body>
        // Any code in here.
      </body>
    </>
  )
}

Home.suppressFirstRenderFlicker = true
Home.getLayout = (page) => <Layout title="Home">{page}</Layout>

export default Home
```

### What are detailed steps to reproduce this?

1. Create a file for your page.
2. Include a `<body>` tag in your JSX/TSX when returning it.

### Run `blitz -v` and paste the output here:

```
OS X El Capitan | darwin-x64 | Node: v14.15.5

blitz: 0.33.1 (global)
blitz: 0.33.1 (local)

  Package manager: yarn 
  System:
    OS: OS X El Capitan 10.11.6
    CPU: (4) x64 Intel(R) Core(TM) i5-5287U CPU @ 2.90GHz
    Memory: 478.16 MB / 8.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.15.5 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.11 - /usr/local/bin/npm
    Watchman: Not Found
  npmPackages:
    @prisma/client: ~2.19 => 2.19.0 
    blitz: 0.33.1 => 0.33.1 
    prisma: ~2.19 => 2.19.0 
    react: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71 
    react-dom: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71 
    typescript: ~4.1 => 4.1.5
```

**Comment by Roesh** [comment]
Able to duplicate on 0.33.0 as well

**Comment by Roesh** [comment]
So, I thought I'd look into this again, and I remember from a create react app type example in ASP.NET Core, there is usually an "index.html" page which contains like a default page, and it does things like display a banner if javascript isn't enabled, informing users to turn it on.

I thought next might have added it, but 
1) I can't seem to find where that file
2) There isn't documentation that we can't use a <body> tag in the blitzpages (or nextpages)

I think next steps would be to determine if this is normal behavior in blitz and nextjs, and if so, update the documentation for both frameworks, and if it isn't leave these issues open until we have a fix

**Comment by flybayer** [comment]
Yes this is by design, `<body>` should be rendered inside `_document.tsx`.

Here's the default one: https://github.com/blitz-js/blitz/blob/canary/packages/generator/templates/app/app/pages/_document.tsx
